packages/core/src/components/Skeleton/Skelton.types.ts

```diff
@@ -0,0 +1,3 @@
+export type SkeletonType = "circle" | "rectangle" | "text";
+
+export type TextSkeletonSize = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "p" | "small";
```

Comment:
What about the 'custom'?

Reply:
The custom is included in SkeletonSizeType, it should also be added here?

Reply:
TextSkeletonSize shouldn't be exported, no?

Reply:
Your right, removed the export